features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/RumFeatureTest.kt

```diff
@@ -1124,6 +1124,49 @@ internal class RumFeatureTest {
         verifyNoInteractions(mockInternalLogger)
     }
 
+    @Test
+    fun `𝕄 handle metric event 𝕎 onReceive()`(
+        @StringForgery fakeMessage: String,
+        forge: Forge
+    ) {
+        // Given
+        val fakeAdditionalProperties = forge.exhaustiveAttributes()
+        testedFeature.onInitialize(appContext.mockInstance)
+        val event = mapOf(
+            "type" to "mobile_metric",
```

Comment:
minor: "mobile_metric" is hardcoded in the tests in several places - perhaps it could be taken from a constant so as to only change in one place in future if the value changed?